plugin/src/main/kotlin/com/vanniktech/maven/publish/MavenPublishBaseExtension.kt

```diff
@@ -80,6 +80,11 @@ abstract class MavenPublishBaseExtension(
     project.tasks.registerDropRepository(buildService)
   }
 
+  @JvmOverloads
+  fun publishToMavenCentral(host: String = "DEFAULT", automaticRelease: Boolean = false) {
```

Comment:
Should I use @ JvmSynthetic to hide from Kotlin?

Reply:
That's a good idea.

We can also leave out the default value for `host` to avoid your concern about conflicting overloads